### PR TITLE
xtl: add version 0.7.4

### PR DESCRIPTION
**Pull request by toge**
Specify library name and version:  **xtl/0.7.4**

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by SSE4 — DISMISSED** [review]

File: recipes/xtl/all/conanfile.py [hunk 1]
```diff
@@ -79,6 +80,7 @@ def _module_file_rel_path(self):
     def package_info(self):
         self.cpp_info.names["cmake_find_package"] = "xtl"
         self.cpp_info.names["cmake_find_package_multi"] = "xtl"
+        self.cpp_info.set_property("cmake_target_name", "xtl")
```
Comment (ericLemanissier) [inline recipes/xtl/all/conanfile.py:83]: I belive the current move is to make these targets name absolute, so xtl::xtl in this case (?)

Comment (prince-chrismc) [reply recipes/xtl/all/conanfile.py:83]: There's usually a delay, so it might be worth holding off but this does benefit some consumers

**Review by prince-chrismc — APPROVED** [review]

File: recipes/xtl/all/conanfile.py (hunk 1 above)
Comment (toge) [reply recipes/xtl/all/conanfile.py:83]: I agree with you.
I made this PR draft.
When CI update conan 1.43.0 or later, I make it "ready for review".

Comment (toge) [reply recipes/xtl/all/conanfile.py]: It's time to fix.

**Review by ericriff — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

**Review by ericLemanissier — APPROVED** [review]

File: recipes/xtl/all/conanfile.py [hunk 2]
```diff
@@ -79,6 +80,7 @@ def _module_file_rel_path(self):
     def package_info(self):
         self.cpp_info.names["cmake_find_package"] = "xtl"
         self.cpp_info.names["cmake_find_package_multi"] = "xtl"
+        self.cpp_info.set_property("cmake_target_name", "xtl::xtl")
```
Comment (SpaceIm) [inline recipes/xtl/all/conanfile.py:83]: Why is it `xtl::xtl`? Official imported target is not namespaced.

**Pull request merged**